### ISSUE TITLE
Jamstix 4 shows black window for 3D Drums view

### ISSUE THREAD

**Issue by taji** [robbert-vdh/yabridge#74]
**Problem description**
Running Jamstix 4 with yabridge (latest).  Host is Reaper for Linux.  The plugin loads and plays back drums fine.  I can see all of the different tabs/screens in the plugin window except the 3D Drum view.  This window is black.  Let me know if you need any other information or assistance. :-)

**What did you expect to happen?**
Should look similar to this:

https://www.rayzoon.com/img2/qs5.jpg

**What actually happened?**
But instead looks like this:

https://drive.google.com/file/d/1V21379c29oVg6HdiR4XFPkFgN3hgYen_/view?usp=sharing

**System information**
- Operating system: Ubuntu Studio 20.04
- wine version: '6.0-rc3 (Staging)'
- Installation method: copies, yabridgectl
- yabridge version: 2.2.1
- yabridgectl version: 2.2.1

**Debug log**
Debug log attached.
[yabridge.log](https://github.com/robbert-vdh/yabridge/files/5743240/yabridge.log)

**Comment by robbert-vdh** [comment]
Have you tried with [DXVK](https://github.com/doitsujin/dxvk)? That usually has much better D3D11 compatibility (and also performance) than WineD3D.

**Comment by taji** [comment]
Will test with DXVK this coming week.  Thanks :-)

**Comment by taji** [comment]
Just finished installing DXVK.  Kit is now visible: 

https://drive.google.com/file/d/1Sh5S6CWakVCl1DxaEC8Mv9Fov8T8Jq0o/view?usp=sharing

Everything seems to be working now.  Audio playback is smooth on my HP I7 laptop.  CPU hovers around 2-4%

In case anyone is interested, I've posted details on the steps I had to take to get Jamstix working here:

http://rayzoon2.com/forum/viewtopic.php?p=49492#49492

Thanks again for the great support! :-)
